src/web/js/trove/chart-lib.js

```diff
@@ -790,7 +795,42 @@
             Object.assign(imageElt, rects[i]); // we should probably not steal *everything*...
             svgRoot.appendChild(imageElt);
           });
+          }
+
+          if (dotChartP) {
+          table.forEach(function (row, i) {
+            // console.log('row', i, '=', row);
+            const rect = rects[i];
```

Comment:
I'm not reviewing this SVG code; I assume from the screenshots that this works adequately enough